config/routes/ibexa_segmentation.yaml

```diff
@@ -0,0 +1,2 @@
+ibexa.segmentation:
```

Comment:
```suggestion
ibexa.ezplatform.segmentation:
```
?

Reply:
@ViniTou Are you sure `ibexa.ezplatform` and not `ibexa.platform`?

Reply:
ibexa.platform

Reply:
I see it has been already done for DAM Connector: https://github.com/ezsystems/ezplatform-ee/blob/master/config/routes/ezplatform_connector_dam.yaml

I also think `ibexa.platform.segmentation` is more appropriate, along with `ibexa_platform_segmentation.yaml` filename. We should fix DAM connector alias and filename to follow this notation.

Reply:
Fixed in 4a63b183

Reply:
Yes, my bad.
fix for DAM:
https://github.com/ezsystems/ezplatform-ee/pull/158